### DISCUSSION TITLE
港澳台番剧搜索出错

### DISCUSSION THREAD

**Discussion by MengNianxiaoyao** [ywmoyue/biliuwp-lite#224] [Q&A] [answered]
![image](https://github.com/ywmoyue/biliuwp-lite/assets/103258286/ab2ae542-9124-4b16-b37d-54ec69caca33.png)
如图，香港和台湾的搜索结果是这样白的，然后点进去番剧也是无法播放的，和在番剧出差的个人页播放番剧的情况一样
[20230724.log](https://github.com/ywmoyue/biliuwp-lite/files/12141897/20230724.log)

**Comment by ywmoyue** [comment, marked as answer]
![image](https://github.com/ywmoyue/biliuwp-lite/assets/28388300/692ca95e-226c-43d0-a5c6-e9c3ed764e31)

视频搜索只能拿到av号，你要用番剧搜索

用非大陆地区的av号直接进入番剧页，在漫游+app上都不行
